api/lib/validate.js

```diff
@@ -166,8 +178,95 @@ function validateOrderbook(orderbook) {
   }
 }
 
+function validateSettings(settings) {
+  if (typeof settings !== 'object') {
+    throw error('Invalid parameter: settings');
```

Comment:
At the point where/if we separate `api` into a library, we don't want it to throw exceptions, but pass errors through the callbacks. This works now because we're catching exceptions in the express middleware (line 39 in express_app.js)